FiveCalls/FiveCalls/IssueDetail.swift

```diff
@@ -40,19 +44,26 @@ struct IssueDetail: View {
                             NavigationLink(value: IssueDetailNavModel(issue: issue, contacts: Array(contacts[contact.number..<contacts.endIndex]))) {
                                 ContactListItem(contact: contact.element, showComplete: store.state.issueCalledOn(issueID: issue.id, contactID: contact.id))
                             }
-                            if contact.number < contacts.count - 1 {
+                            if contact.number < contacts.count - 1 || contact.number == contacts.count - 1 && !vacantAreas.isEmpty {
```

Comment:
I realize this is a stretch but this only handles the case of a single vacancy, right? We don't have to handle the case for multiple vacancies since it's so rare, but I think it would help to better describe what this is doing ("either are not done with the contact list or it's the last contact and we have a vacancy to show")